app/views/reviews/index.html.slim

```diff
@@ -2,45 +2,97 @@
 - content_for :page_heading, "My Reviews"
 
 - if @reviews.size > 0
+  .row
+    .col-xs-6.myreviews-stats-col1
+      .container-fluid#total-reviews-container
+        h3 Total Reviews Written
+        h1#myreviews-stats-number =@reviews.count
+    .col-xs-6.myreviews-stats-col2
+      .container-fluid#average-professor-ratings-container
+        h3 Average Professor Ratings
+        h1#myreviews-stats-number =@avg_prof_rating
+  .container-fluid#addreview-container = link_to new_review_path
+    .addreview
+      .row
+        .col-xs-12
+          h4#addreview-label Write Another Review
+        .col-xs-12
+          span.glyphicon.glyphicon-plus
+
+  br
   - @semesters.each do |semester|
     .row
       .col-xs-12
-        h3= "#{semester.season} #{semester.year}"
+        h2#semester-header = "#{semester.season} #{semester.year}"
     - @reviews_map[semester.number].each do |review|
-      .row
-        .col-xs-2
-          = link_to "#{review.course.mnemonic_number}", review.course
-        .col-xs-3
-          = link_to review.professor.full_name, review.professor
-        .col-xs-2
-          = link_to "Edit Review", edit_review_path(review)
-        .col-xs-5
-          a href="#{course_path(review.course_id, :p => review.professor_id)}"
-            span Go To Class 
-            span.glyphicon.glyphicon-arrow-right
+      div.myreview-header
+        .row.myreview-header-row
+          .col-xs-2
+            h3= link_to "#{review.course.mnemonic_number}", review.course
+          .col-xs-2
+            = link_to review.professor.full_name, review.professor
+          .col-xs-3
+            a href="#{course_path(review.course_id, :p => review.professor_id)}"
+              span Go To Class 
+              span.glyphicon.glyphicon-arrow-right
+          .col-xs-5
+            div.edit-review
+              = link_to "Edit Review", edit_review_path(review)
+      .container-fluid.myreview-container
```

Comment:
This would be a great time to define a set style for all reviews. Think this part.

<img width="977" alt="review" src="https://cloud.githubusercontent.com/assets/5743276/23089566/cac2c1ca-f558-11e6-8977-b259451ac33c.png">
So now is a good time to define the styles that we can use BOTH on the course page and on the my reviews page! Try to use the guidelines in the CSS guide I linked above to come up with the element names, etc.